README.md

```diff
@@ -20,98 +17,20 @@ To run this CLI application
-## Quickstart
-
-To get started with Scaffold-ETH 2, follow the steps below:
-
-1. Install from NPM Registry and follow the CLI instructions.
-
-```
-npx create-eth@latest
-```
-
-> 💬 Hint: If you choose Foundry as solidity framework in the CLI, you'll also need Foundryup installed in your machine. Checkout: [getfoundry.sh](https://getfoundry.sh)
-
-2. Run a local network in the first terminal:
-
-```
-yarn chain
-```
-
-This command starts a local Ethereum network using Hardhat or Foundry, depending on which one you selected in the CLI. The network runs on your local machine and can be used for testing and development. You can customize the network configuration in:
-
-- `packages/hardhat/hardhat.config.ts` if you have Hardhat as solidity framework.
-- `packages/foundry/foundry.toml` if you have Foundry as solidity framework.
-
-3. On a second terminal, deploy the test contract:
-
-```
-yarn deploy
-```
-
-This command deploys a test smart contract to the local network. The contract can be modified to suit your needs. Is located in:
-
-- Hardhat => `packages/hardhat/contracts`
-- Foundry => `packages/foundry/contracts`
-
-The `yarn deploy` command uses a deploy script to deploy the contract to the network. You can customize it. Is located in:
-
-- Hardhat => `packages/hardhat/deploy`
-- Foundry => `packages/foundry/script`
-
-4. On a third terminal, start your NextJS app:
-
-```
-yarn start
-```
-
-Visit your app on: `http://localhost:3000`. You can interact with your smart contract using the `Debug Contracts` page. You can tweak the app config in `packages/nextjs/scaffold.config.ts`.
-
-Run smart contract test with `yarn hardhat:test` or `yarn foundry:test` depending of your solidity framework.
-
-**What's next**:
-
-- Edit your smart contract `YourContract.sol` in `packages/hardhat/contracts`
-- Edit your frontend homepage at `packages/nextjs/app/page.tsx`. For guidance on [routing](https://nextjs.org/docs/app/building-your-application/routing/defining-routes) and configuring [pages/layouts](https://nextjs.org/docs/app/building-your-application/routing/pages-and-layouts) checkout the Next.js documentation.
-- Edit your deployment scripts in `packages/hardhat/deploy`
-- Edit your smart contract test in: `packages/hardhat/test`. To run test use `yarn hardhat:test`
-
-## Documentation
-
-Visit our [docs](https://docs.scaffoldeth.io) to learn how to start building with Scaffold-ETH 2.
-
-To know more about its features, check out our [website](https://scaffoldeth.io).
-
-## Contributing to Scaffold-ETH 2
-
-We welcome contributions to Scaffold-ETH 2!
-
-Please see [CONTRIBUTING.MD](https://github.com/scaffold-eth/scaffold-eth-2/blob/main/CONTRIBUTING.md) for more information and guidelines for contributing to Scaffold-ETH 2.
+## MVP
+- ETT CLI enables users to see the full scope of the tech tree
+- They can see clearly which ones are unlocked and which ones are still locked
+- It only allows them to interact with the challenges that are unlocked
```

Comment:
You might consider re-scoping the MVP to not worry about handling all the logic necessary for preventing a user from attempting a challenge. Would permission all be handled by the CLI or would the logic be server side to prevent submission based on a user's existing status in the database?

Reply:
I am envisioning that both CLI and backend will have logic in place that prevents users from submitting prematurely. One caveat is that at some point we may allow users to skip ahead if they can prove their skill. I will add a new section with wandering ideas like this.

Reply:
I agree with Matt. The MVP probably doesn't really care about whether or not the user is gated from harder challenges. The MVP is about just getting the proper challenges set up to get people through the door and giving feedback. We want their feedback on the UX of learning things with this UX (CLI or not) firstly. The gating aspect can be an add on. That said, if it's not that hard to do, then it probs can just be in the MVP.